src/navability/entities/Variable.jl

```diff
@@ -8,6 +8,13 @@ DFG_VERSION = "0.17.1";
     POSE2
 end
 
+@enum QueryDetail begin
```

Comment:
Neat :+1: Haven't used one of these before.

Reply:
They suck. Need to discuss with Dehann best way to handle this.